README.md

```diff
@@ -53,6 +53,15 @@ Capital Framework is our modular front end framework. New projects should start
 - **Willingness**: We are willing to help each other with anything that is good for the team
 - **Excellence**: We believe that excellence is defined by a commitment to and practice of constant learning and improvement
 
+## Our Front End Web Development Values
```

Comment:
Trivial "Front-end web development values" (and the prior heading should also be sentence-cased).